std/algorithm/setops.d

```diff
@@ -611,6 +593,27 @@ void largestPartialIntersection
             sorted);
 }
 
+///
```

Comment:
The paragraph below the example references the example, but this change moves the example below the text. I think in these cases we need to either leave the example in the doc comment, or restructure the doc text to work with the new layout.